src/main/resources/entity-types/orders/composite_purchase_order_lines.json5

```diff
@@ -83,6 +88,7 @@
       alias: 'rates',
       type: 'entity-type',
       id: '770a383e-c6cb-412b-abc3-6fa3c8a8795c', // simple_user_details
+      essentialOnly: true,
```

Comment:
Would this do anything? This entity type id doesn't correspond to simple_user_details

Reply:
Correct. Added by mistake. Thanks @bvsharp